templates/publish.yml

```diff
@@ -19,6 +19,10 @@ spec:
       default: ""
     team:
       default: uds
+    mirror:
+      default: "true"
+    mirror-project-id:
+      default: "11"
```

Comment:
```suggestion
```

IMO we should not have this default here since it is env specific and not really a default for all envs.